bossggle.js

```diff
@@ -8,21 +8,41 @@ setupListeners();
 
 $(document).ready(function() {
 	$("#submit").on("click", function() {
-		givePoints();
-		addWord();
-		playWord = ""
-		clearWord();
-
+		//playWord = playWord.toLowerCase();
+		if  (playWord.length < 3) {
+			Materialize.toast("Your word must have at least 3 letters", 3000);
+			clearWord();		
+		} else if (!Word_List.isInList(playWord)) {
+			Materialize.toast("That's not a word we know :-(, try again!", 3000);
+		} else {
+			givePoints();
+			addWord();
+			clearWord();
+		}
 	});
 	$("#clear").on("click", function() {
-		playWord = "";
 		clearWord();
 	});
 	$("#reset").on("click", function() {
-		playWord = "";
-		setupLetters();
+		setupLetters(true);
 		clearWord();
-	});
+    });
+
+	// start button
```

Comment:
Please fix this formatting. I might be reading this wrong on Github, but it appears that your start button logic is indented quite a bit though it does not logically live under any particular piece of logic. Then, your right column game is outdented all the way... it might want to live underneath something else.